### ISSUE TITLE
Bug in PFRejuv Store Handing

### ISSUE THREAD

**Issue by null-a** [probmods/webppl#149]
I expect [this test](https://github.com/null-a/webppl/commit/c20c5d6ed04dc35aeeca973035e33dc400a6f6cf) to always return `{ 'true': 1 }` but PF with rejuvenation is a little off:

```
Expected: { true: 1 }
Actual: { false: 0.02580645161290322, true: 0.9741935483870968 }
✖ PFRj - store2

Expected: { true: 1 }
Actual: { false: 0.019801980198019802, true: 0.9801980198019803 }
✖ PFRjAsMH - store2
```

I guess this is a problem with the store handling, but I've not tracked it down yet. I don't think my work-in-progress refactoring of MH & SMC (#64) suffers from this, though I'm not sure we'd want to wait for that to be merged before having a fix for this.

**Comment by stuhlmueller** [comment]
My hunch would be to work on the refactoring instead of tracking down the bug, unless you think it won't take long to fix it, or that it will take multiple months for the refactoring to make it into dev.

**Comment by null-a** [comment]
Closed by #169.
